contributions/catalog/utils/requestutil.py

```diff
@@ -117,4 +117,19 @@ def request_required_capability_list(headers):
                                        "capabilityId": capability_id}
             required_capability_list.append(tmp_required_capability)
 
-    return required_capability_list
\ No newline at end of file
+    return required_capability_list
+
+"""
+request building blocks list
+"""
+def request_buildingblocks(headers):
+    request_url = cfg.CONTRIBUTION_BUILDING_BLOCK_URL + "/building-blocks"
+    result = requests.get(request_url, headers=headers)
+
+    # create the list of required capabilities
+    if result.status_code == 200:
+        # check if the login id is in reviewer list
+        result_str = result.content.decode('utf-8').replace('\n', '')
+        building_block_list_json = json.loads(result_str)
```

Comment:
building_block_list_json is inside an if condition. If the condition not satisfied, it will have an error. Maybe use a try except block? Or define building_block_list=[] outside if?

Reply:
This will not happen but I will add it.